lib/slas.js

```diff
@@ -1,18 +1,22 @@
 const fetch = require('node-fetch');
```

Comment:
Thanks very useful changes in that module. Appreciate you fixing my sloppy copy and paste  errors